usagov/usagov-2021
# USAGOV-1942-icon-all-benefits

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1942

## Description

Update CSS to use life event ID in CSS class instead of title. Adds a selector to display icons for an "All Benefits" life event

## Type of Changes


![image](https://github.com/user-attachments/assets/04f74129-7065-4f20-a024-45aee8e0a321)


## Testing Instructions



### Change Requirements


- [] Requires New Content

### Validation Steps

1. Import configuration to update life events grid view
2. Create or publish a life event with ID="all_benefits", with a search title and short description field. Language=en
3. Create or publish a life event with ID="es_all_benefits", with a search title and short description field. Language=es
4. On English page, `/beneft-finder' page, confirm a new tile displays in the grid with an icon and the title and short description from step 2 above.
5.  On Spanish page, `/es/buscador-beneficios' page, confirm a new tile displays in the grid with an icon and the title and short description from step 3 above.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
